homeassistant/components/roborock/select.py

```diff
@@ -26,7 +22,8 @@ class RoborockSelectDescriptionMixin:
 
     api_command: RoborockCommand
     value_fn: Callable[[Status], str]
-    options_lambda: Callable[[str], list[int]]
+    options_lambda: Callable[[Status], list[str]]
```

Comment:
As i'm reading this, quite difficult to know what `options_lambda` vs `option_lamda` represent. Perhaps a comment or another name here can help.  Is this like: current option vs available options?

Reply:
Renamed to options lambda and parameter_lamba and gave some extra commenting.